packages/story-editor/src/app/font/actions/useLoadFontFiles.js

```diff
@@ -42,7 +42,12 @@ function useLoadFontFiles() {
 
     switch (service) {
       case 'fonts.google.com':
-        await loadStylesheet(getGoogleFontURL([{ family, variants }], 'auto'));
+        await loadStylesheet(
+          getGoogleFontURL([{ family, variants }], 'auto'),
+          elementId
+        ).catch(() => {
+          return;
+        });
```

Comment:
Since we're using async/await here, we'd need to wrap this in a `try ... catch`. It's not a Promise where one could use `.catch()`.

```suggestion
				try {
          await loadStylesheet(
         	  getGoogleFontURL([{ family, variants }], 'auto'),
         	  elementId
          );
        } catch {
        	// Do nothing.
      	}
```